.github/workflows/pythonapp.yml

```diff
@@ -8,147 +8,155 @@ jobs:
+      - name: Run checker tests
+        run: |
+          coverage run --append -m pytest -v test/test_checkers.py
+      - name: Run cli tests
+        run: |
+          coverage run --append -m unittest test.test_cli
+        continue-on-error: true
+      - name: Run csv2cve tests
+        run: |
+          coverage run --append -m unittest test.test_csv2cve
+      - name: Run cvedb tests
+        run: |
+          coverage run --append -m unittest test.test_cvedb
+      - name: Run output engine tests
+        run: |
+          coverage run --append -m unittest test.test_output_engine
+      - name: Run extract tests
+        run: |
+          coverage run --append -m unittest test.test_extract
+      - name: Run file tests
+        run: |
+          coverage run --append -m unittest test.test_file
+      - name: Run json tests
+        run: |
+          coverage run --append -m unittest test.test_json
+      - name: Run scanner tests
+        run: |
+          coverage run --append -m pytest -v test/test_scanner.py
+        continue-on-error: true
+      - name: Run strings tests
+        run: |
+          coverage run --append -m unittest test.test_strings
+      - name: Run util tests
+        run: |
+          coverage run --append -m unittest test.test_util
+      - name: Create xml file of coverage report
+        run: |
+          coverage xml -o win_cov_report.xml
+      - name: upload code coverage to codecov
+        uses: codecov/codecov-action@v1
+        with:
+          file: ./win_cov_report.xml
+          flags: wintests
+          name: codecov-umbrella
+          fail_ci_if_error: false
 
   long_tests:
     name: Long tests on python3.8
     runs-on: ubuntu-latest
     timeout-minutes: 30
     env:
       ACTIONS: 1
-      LONG_TESTS: 1
     steps:
-    - uses: actions/checkout@v2
-    - name: Set up Python
-      uses: actions/setup-python@v1
-      with:
-        python-version: 3.8
-    - name: Install dependencies
-      run: |
-        python -m pip install --upgrade pip
-        pip install -r requirements.txt
-    - name: try single cli run of tool
-      run: |
-        python -m cve_bin_tool.cli test/binaries
-    - name: Run long tests
-      run: |
-        pip install coverage
-        coverage run --append setup.py test
-        coverage run --append -m pytest -v test/test_scanner.py test/test_checkers.py
-        coverage xml -o py38_cov_report.xml
-    - name: upload code coverage to codecov
-      uses: codecov/codecov-action@v1
-      with:
-        file: ./py38_cov_report.xml
-        flags: longtests
-        name: codecov-umbrella
-        fail_ci_if_error: false
+      - uses: actions/checkout@v2
+      - name: Set up Python
+        uses: actions/setup-python@v1
+        with:
+          python-version: 3.8
+      - uses: technote-space/get-diff-action@v1
+        with:
+          PREFIX_FILTER: |
+            cve_bin_tool/checkers
+            test/test_scanner
```

Comment:
There are long tests that aren't associated with a checker  I think at the moment they're mostly in cvedb.py and json.py is nothing but a single long test.  I think we'll need a bit more finesse here to make this work without resulting in regressions due to failing to test.